ubuntu/debian/control

```diff
@@ -47,6 +47,9 @@ Depends: libgz-cmake3-dev,
          libgz-utils2-cli-dev,
          libgz-utils2-dev,
          libsdformat13-dev,
+         gz-launch6-cli
+         gz-sim7-cli,
```

Comment:
can you add `gz-plugin2-cli` as well?

Reply:
sure [2be21fe](https://github.com/gazebo-release/gz-garden-release/pull/16/commits/2be21fe7181d1491b7eefaa7c71e53304cd3bc35).